src/data/links.yaml

```diff
@@ -1439,3 +1443,46 @@ links:
       organization holds retailers accountable by monitoring how offsets are
       transacted and advertised in the retail market, protecting both the buyer
       and the seller.
+
+  - title: TenTree
+    url: https://www.tentree.ca/
+    featured: true
+    categories:
+      - clothing
+    tags: [clothing]
+    description: >-
+      Ten tree is a brand that actively promotes environmental stewardship. They have a wide variety of comfortable clothes and accessories and with each purchase, they pledge to plant 10 trees.
+
+  - title: Arvin  Goods
+    url: https://www.arvingoods.com/
+    featured: true
+    categories: 
+      - clothing
+    tags: [clothing]
+    description: >-
+      Brand that makes fashionable socks through a sustainable supply chain process they have developed. They have removed excess water from their manufacturing process - leaving more water left for communities to drink!
+
+  - title: Siizu
+    url: https://siizu.com/
+    categories:
+      - clothing
+    tags: [clothing]
+    description: >-
+      A mininalistic-styled brand that creates clothing with organic and natural fabrics. They use an ethical manufacturing process by having their factories close to each other and their shipping only uses recyclable packaging.
+
+  - title: For Days
+    url: https://www.fordays.com/
+    categories:
+      - clothing
+    tags: [clothing]
+    description: >-
+        Subsciption service that provides clothing with a zero-waste approach. When you sign up you purchase a membership and recieve an organic cotton shirt -  each time you want to purchase another article you send them back any old item to swap them out.
+
+  - title: Kotn
+    url: https://kotn.com/
+    categories:
+      - clothing
+    tags: [clothing]
+    description: >-
+        A canadian company that uses organic cotton to produce high quality clothes. They source their cotton from farmers in Egypt.
```

Comment:
This sounds like the clothes are flying around rather a lot, and cotton as mentioned is awful. Let's take this one out.